src/android_components.py

```diff
@@ -36,6 +36,54 @@ def update_gv_version(ac_repo, old_gv_version, new_gv_version, branch, channel,
     ac_repo.update_file(contents.path, f"Update GeckoView ({channel.capitalize()}) to {new_gv_version}.",
                      new_content, contents.sha, branch=branch, author=author)
 
+#
+# Update GeckoView Nightly on A-C master. This is a bit of a special
+# case since it doesn't care about release branches. It can probably
+# be refactored to share more code with update_geckoview() though.
+#
+
+def update_geckoview_nightly(ac_repo, fenix_repo, author, debug):
+    try:
+        channel = "nightly"
+        release_branch_name = "master"
+
+        current_gv_version = get_current_gv_version(ac_repo, release_branch_name, channel)
+        current_gv_major_version = current_gv_version.split(".")[0]
+        latest_gv_version = get_latest_gv_version(current_gv_major_version, channel)
+
+        #
+        # Create a new branch for this update
+        #
+
+        release_branch = ac_repo.get_branch(release_branch_name)
+        print(f"{ts()} Last commit on {release_branch_name} is {release_branch.commit.sha}")
+
+        pr_branch_name = f"GV-Nightly-{latest_gv_version}"
+        ac_repo.create_git_ref(ref=f"refs/heads/{pr_branch_name}", sha=release_branch.commit.sha)
+        print(f"{ts()} Created branch {pr_branch_name} on {release_branch.commit.sha}")
+
+        #
+        # Update buildSrc/src/main/java/Gecko.kt
+        #
+
+        print(f"{ts()} Updating buildSrc/src/main/java/Gecko.kt")
+        update_gv_version(ac_repo, current_gv_version, latest_gv_version, pr_branch_name, channel, author)
+
+        #
+        # Create the pull request
+        #
+
+        print(f"{ts()} Creating pull request")
+        pr = ac_repo.create_pull(title=f"GeckoView update ({channel.capitalize()}) {latest_gv_version}.",
+                                 body=f"This (automated) patch updates GV {channel.capitalize()} on master to {latest_gv_version}.",
+                                 head=pr_branch_name, base=release_branch_name)
+        pr.add_to_labels("🛬 needs landing")
```

Comment:
nit: This label is not needed for bot PRs.

Reply:
Removed from this PR. I also filed an additional PR to also remove it from the Beta task.